staticaddr/withdraw/manager.go

```diff
@@ -726,105 +739,87 @@ func toOutpoints(deposits []*deposit.Deposit) []wire.OutPoint {
 // signMusig2Tx adds the server nonces to the musig2 sessions and signs the
 // transaction.
 func (m *Manager) signMusig2Tx(ctx context.Context,
-	prevOutFetcher *txscript.MultiPrevOutFetcher, outpoints []wire.OutPoint,
+	prevOutFetcher *txscript.MultiPrevOutFetcher,
```

Comment:
Add tests for signMusig2Tx.

When sigInfo is missing an entry for one of the deposits, the call should error (no partial return). Build a dummy tx with two inputs, provide sessions for both, but only one SigningInfo entry; expect error.

When sigInfo has all expected keys but one maps to the wrong index (mismatched depositsToIdx), ensure it errors.